src/createContainer.ts

```diff
@@ -21,6 +22,7 @@ type AnyFunction = (...args: any[]) => any;
 
 export const createContainer = <State, Update extends AnyFunction, Props>(
   useValue: (props: Props) => readonly [State, Update],
+  legacyMode = false,
```

Comment:
I would suggest introducing a breaking change and inverting the flag.
```suggestion
  concurrentMode = false,
```